IBPSA/Examples/Tutorial/SimpleHouse/SimpleHouse4.mo

```diff
@@ -0,0 +1,133 @@
+within IBPSA.Examples.Tutorial.SimpleHouse;
+model SimpleHouse4 "Heating model"
+  extends SimpleHouse3;
+
+  parameter Modelica.Units.SI.HeatFlowRate QHea_nominal=3000
+    "Nominal capacity of heating system";
+  parameter Modelica.Units.SI.MassFlowRate mWat_flow_nominal=0.1
+    "Nominal mass flow rate for water loop";
+  parameter Boolean constantSourceHeater=true
+    "To enable/disable the connection between the constant source and heater";
+
+  IBPSA.Fluid.HeatExchangers.Radiators.RadiatorEN442_2 rad(
+    redeclare package Medium = MediumWater,
+    T_a_nominal=333.15,
+    T_b_nominal=313.15,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    allowFlowReversal=false,
+    Q_flow_nominal=3000)                          "Radiator"
+    annotation (Placement(transformation(extent={{110,-110},{130,-90}})));
+  IBPSA.Fluid.HeatExchangers.HeaterCooler_u heaWat(
+    redeclare package Medium = MediumWater,
+    m_flow_nominal=mWat_flow_nominal,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.SteadyState,
+    allowFlowReversal=false,
+    dp_nominal=5000,
+    Q_flow_nominal=QHea_nominal) "Heater for water circuit"
+    annotation (Placement(transformation(extent={{60,-110},{80,-90}})));
+  IBPSA.Fluid.Movers.FlowControlled_m_flow pump(
+    redeclare package Medium = MediumWater,
+    use_inputFilter=false,
+    m_flow_nominal=mWat_flow_nominal,
+    energyDynamics=Modelica.Fluid.Types.Dynamics.SteadyState,
+    allowFlowReversal=false,
+    nominalValuesDefineDefaultPressureCurve=true,
+    inputType=IBPSA.Fluid.Types.InputType.Constant)
+                                         "Pump"
+    annotation (Placement(transformation(extent={{110,-180},{90,-160}})));
+  IBPSA.Fluid.Sources.Boundary_pT bouWat(redeclare package Medium = MediumWater, nPorts=1)
+    "Pressure bound for water circuit" annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        origin={10,-170})));
+  Modelica.Blocks.Sources.Constant const(k=1)
+    annotation (Placement(transformation(extent={{80,-80},{60,-60}})));
+equation
+  connect(heaWat.port_b,rad. port_a) annotation (Line(points={{80,-100},{110,-100}},
+                       color={0,127,255}));
+  connect(rad.port_b,pump. port_a) annotation (Line(points={{130,-100},{148,-100},
+          {148,-170},{110,-170}},color={0,127,255}));
+  connect(heaWat.port_a,pump. port_b) annotation (Line(points={{60,-100},{49.75,
+          -100},{49.75,-170},{90,-170}},  color={0,127,255}));
+  connect(rad.heatPortCon, zone.heatPort) annotation (Line(points={{118,-92.8},
+          {118,140},{110,140}},color={191,0,0}));
+  connect(rad.heatPortRad, walCap.port) annotation (Line(points={{122,-92.8},{122,
+          -30},{132,-30},{132,1.77636e-15},{140,1.77636e-15}}, color={191,0,0}));
+  if constantSourceHeater then
+    connect(const.y, heaWat.u) annotation (Line(points={{59,-70},{50,-70},{50,-94},
+          {58,-94}}, color={0,0,127}));
+  end if;
```

Comment:
This line does not show in my Open Modelica ( v1.22.0-dev)
![image](https://github.com/ibpsa/modelica-ibpsa/assets/22442514/f20e3dad-94e2-4f6a-962c-c5efe6b02dbb)

Reply:
This seems to be a bug in OpenModelica (Dymola always shows the connection, even if `use_constantHeater=false`. However, when the users model this themselves, they can just remove the `Constant` component (this is a bad type of control after all), so they won't have this problem.

But I understand that it might be confusing when they check the 'model solution'. I could use a switch, but that might also confuse them (since only one switch input will have a connection in `SimpleHouse4`).

What do you think?

Reply:
Mhm, that is true. I will raise this Bug to OM. Maybe they can fix it rather quickly. I would leave it as it is currently, then.

Reply:
I raised the issue: https://github.com/OpenModelica/OpenModelica/issues/11202